internal/resolution/variable_sources/entity/bundle_entity_test.go

```diff
@@ -243,7 +243,7 @@ var _ = Describe("BundleEntity", func() {
 	Describe("BundlePath", func() {
 		It("should return the bundle channel properties if present", func() {
 			entity := input.NewEntity("operatorhub/prometheus/0.14.0", map[string]string{
-				"olm.bundle.path": "bundle.io/path/to/bundle",
+				"olm.bundle.path": `"bundle.io/path/to/bundle"`,
```

Comment:
Is this a change in semantic?

Reply:
Again part of the v0 CatalogSource integration that the first commit reverts. cc: @ankitathomas @perdasilva 

So looks like bundle_entity.go and bundle_entity_test.go gets affected by the revert. I'm happy to just...revert..the...revert for these two files, but it's probably prudent to have you guys chime in with the context for these kind of changes first so that I am sure of what I'm doing :)